docs/project/glossary.md

```diff
@@ -232,34 +127,58 @@ officially updated since .NET Framework 2.0.
 
 ### Mono
 
-Mono is an open source alternative to the .NET Framework. Mono started around
+[Mono][mono] is an open source alternative to the .NET Framework. Mono started around
 the same time the .NET Framework was first released. Since Microsoft didn't
-release Rotor as open source, Mono was forced to start from scratch and is thus
+release Rotor as open source, Mono was forced to start from scratch and was thus
 a complete re-implementation of the .NET Framework with no shared code.
 
-When .NET Core was released under the MIT license, Microsoft also released large
-chunks of the .NET Framework under the MIT license as well, which can be found
-[here][referencesource]. This enabled the Mono community to use the same code
-the .NET Framework uses in order to close gaps and avoid behavioral differences.
+Today, the [Mono runtime](https://github.com/dotnet/runtime/tree/master/src/mono) is part
```

Comment:
```suggestion
Today, the [Mono VM](https://github.com/dotnet/runtime/tree/master/src/mono) is part
of the unified One .NET platform. It is optimized embeddable VM used primary for mobile, browser or gaming like scenarios.

Reply:
> embeddable 

I think I would need to define what it means to be "embeddable" somewhere.

> gaming 

I believe that the only runtime optimized for gaming is Unity. I will mention Unity here. Outside Unity, I do not think we really heave runtime optimized for gaming. Games outside Unity use every other .NET runtime out there, including niche runtimes like BRUTE.

Reply:
I cannot list specifics but we have been offering this to our paying customers as official SDKs so Unity is only one of the offerings.